src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -631,7 +688,8 @@ For example:
 ```JSON
 {
    "EchoTime1": 0.00600,
-   "EchoTime2": 0.00746
+   "EchoTime2": 0.00746,
+   "B0FieldIdentifier": "phasediff_fmap0"
```

Comment:
I noticed that the example `B0FieldIdentifier` values in this PR are fairly standardized. @oesteban do you want a specific vocabulary to be put in place?

Reply:
I cannot think of a reason to establish a vocabulary for this. I think giving recommendations could be a possibility (along these lines I suggested somewhere using some DICOM field for inspiration - I should go back to that).

For instance, it could be convenient to avoid identifiers that would not be valid BIDS labels down the line (you can imagine a workflow making this field an entity inside BIDS-Derivatives). And yet, you can just strip disallowed chars out if you were to do that. So perhaps, do not use invalid BIDS-label chars for differentiating ids? E.g., one fieldmap set having `phasediff0` and another fieldmap set with `phasediff_0` (or maybe worse, `phasediff0_`). And still this feels a bit twisted for naming.